a5py/routines/runmixin.py

```diff
@@ -1420,7 +1470,9 @@ def plotwall_3dstill(self, wallmesh=None, points=None, orbit=None,
 
     def plotwall_3dinteractive(self, wallmesh=None, *args, points=None,
                                orbit=None, data=None, log=False,
-                               cpos=None, cfoc=None, cang=None, **kwargs):
+                               cpos=None, cfoc=None, cang=None,
+                               particle_filter = None, wall_filter = None,
```

Comment:
No spaces. Also here as in other places I suggest changing particle_filter to ids. Maybe wall_filter could then be wall_ids?